### PR TITLE
Fix case. Php < PHP

### PR DESCRIPTION
**Pull request by hstsethi**
## What does this PR do?
Fixes case of "PHP" in Hindi booklist.

### Why is this valuable (or not)?

I believe it is. 

### How do we know it's really free?

Not applicable

### For book lists, is it a book? For course lists, is it a course? etc.

Not applicable

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Ajay-singh1 — APPROVED** [review]
LGTM!Wait for Eric.

**Comment by hstsethi** [comment]
> agreed, but the rest of the titles don't match

I am sorry, but I don't understand. Do you mean the heading?

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -62,7 +62,7 @@
 * [ डाटा कयनकेशन एंड कंयटर नेटवक ](https://www.aicte-india.org/sites/default/files/HINDI_BOOKS/BOOK%203.pdf) -  ई.हरश दाधीच, ई.वकास माथर (PDF)
 
 
-### Php
+### PHP
 
-* [Php \| Hindi](https://www.learnhindituts.com/php) - LearnHindiTuts.com
-* [Php Hindi Tutorial \| Hindi](http://tutorialsroot.com/php/index.html) - TutorialsRoot.com
+* [PHP \| Hindi](https://www.learnhindituts.com/php) - LearnHindiTuts.com
```
Comment (eshellman) [inline books/free-programming-books-hi.md:67]: title appears to be "PHP Tutorials In Hindi"

File: books/free-programming-books-hi.md [hunk 2]
```diff
@@ -62,7 +62,7 @@
...
+* [PHP Hindi Tutorial \| Hindi](http://tutorialsroot.com/php/index.html) - TutorialsRoot.com
```
Comment (eshellman) [inline books/free-programming-books-hi.md:68]: title appears to be "PHP in Hindi Tutorial"

**Review by eshellman — CHANGES_REQUESTED** [review]
Don't invent titles! Use the title given by the resource.

**Comment by hstsethi** [comment]
> Don't invent titles! Use the title given by the resource.

I am really sorry for the inconvenience caused. I have followed your suggestions. Please review.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**